.github/workflows/tox.yml

```diff
@@ -12,12 +12,12 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
```

Comment:
Shouldn't it be 22.04 as that is what our RIOT docker is currently at?

Reply:
To test the python versions we want to test (it seems) we need that ubuntu version. You can check https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json